src/CoinbaseSmartWallet.sol

```diff
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.23;
 
-import {Receiver} from "solady/src/accounts/Receiver.sol";
-import {UUPSUpgradeable} from "solady/src/utils/UUPSUpgradeable.sol";
-import {SignatureCheckerLib} from "solady/src/utils/SignatureCheckerLib.sol";
```

Comment:
Solady updated something such that default forge remapping now works and this can be removed